### PR TITLE
Account for linter complains about terminology

### PR DESCRIPTION
**Pull request by MrMCake**
# Change

- Account for linter complains about terminology. For example 
  - 'ID' instead of 'id'
  - 'Repository' instead of 'repo'
  - 'API' instead of api
- Removed storage account `deploy.json`

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

File: arm/Microsoft.DesktopVirtualization/hostpools/readme.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ The below parameter object is converted to an in-line string when handed over to
         "osDiskType": "StandardSSD_LRS",
         "useManagedDisks": true,
         "vmSize": {
-            "id": "Standard_D2s_v3",
+            "ID": "Standard_D2s_v3",
```
Comment (MariusStorhaug) [inline arm/Microsoft.DesktopVirtualization/hostpools/readme.md:92]: are we sure this will work?

File: docs/wiki/GettingStarted.md [hunk 2]
```diff
@@ -124,9 +124,9 @@ In case you want to not only leverage the module templates but actually re-use t
 
 | Secret Name | Example | Description |
 | - | - | - |
-| `ARM_MGMTGROUP_ID` | `de33a0e7-64d9-4a94-8fe9-b018cedf1e05` | The ID of the management group to test deploy modules of that level in. |
-| `ARM_SUBSCRIPTION_ID` | `d0312b25-9160-4550-914f-8738d9b5caf5` | The ID of the subscription to test deploy modules of that level in. |
-| `ARM_TENANT_ID` | `9734cec9-4384-445b-bbb6-767e7be6e5ec` | The ID of the tenant to test deploy modules of that level in. |
+| `ARM_MGMTGROUP_ID` | `de33a0e7-64d9-4a94-8fe9-b018cedf1e05` | The resource ID of the management group to test deploy modules of that level in. |
+| `ARM_SUBSCRIPTION_ID` | `d0312b25-9160-4550-914f-8738d9b5caf5` | The resource ID of the subscription to test deploy modules of that level in. |
+| `ARM_TENANT_ID` | `9734cec9-4384-445b-bbb6-767e7be6e5ec` | The resource ID of the tenant to test deploy modules of that level in. |
```
Comment (MariusStorhaug) [inline docs/wiki/GettingStarted.md:129]:
```suggestion
| `ARM_MGMTGROUP_ID` | `de33a0e7-64d9-4a94-8fe9-b018cedf1e05` | The group ID of the management group to test deploy modules of that level in. |
| `ARM_SUBSCRIPTION_ID` | `d0312b25-9160-4550-914f-8738d9b5caf5` | The subscription ID of the subscription to test deploy modules of that level in. |
| `ARM_TENANT_ID` | `9734cec9-4384-445b-bbb6-767e7be6e5ec` | The tenant ID of the tenant to test deploy modules of that level in. |
```

File: constructs/Microsoft.Management/managementGroup-structure/readme.md [hunk 3]
```diff
@@ -63,9 +63,9 @@ Describes the Management groups to be created. Each management group is represen
 
 | Parameter Name | Type | Default Value | Possible values | Description |
 | :-             | :-   | :-            | :-              | :-          |
-| `name` | string | | | Mandatory. The ID of the Management group |
+| `name` | string | | | Mandatory. The resource ID of the Management group |
```
Comment (MariusStorhaug) [inline constructs/Microsoft.Management/managementGroup-structure/readme.md:66]:
```suggestion
| `name` | string | | | Required. The group ID of the Management group |
```

File: constructs/Microsoft.Management/managementGroup-structure/readme.md [hunk 4]
```diff
@@ -116,8 +116,8 @@ This template is using a **Tenant level deployment**, meaning the user/principal
 > **Management Group Contributor** at the top management group that needs to be managed
 >
 >> Consider using the following script:<br>
->> `$PrincipalID = "<The id of the identity here>"`<br>
->> `$TopMGID = "<The id of the management group here>"`<br>
+>> `$PrincipalID = "<The resource ID of the identity here>"`<br>
+>> `$TopMGID = "<The resource ID of the management group here>"`<br>
```
Comment (MariusStorhaug) [inline constructs/Microsoft.Management/managementGroup-structure/readme.md:120]:
```suggestion
>> `$PrincipalID = "<The object ID of the identity here>"`<br>
>> `$TopMGID = "<The group ID of the management group here>"`<br>
```

File: arm/Microsoft.Management/managementGroups/readme.md [hunk 5]
```diff
@@ -18,9 +18,9 @@ This module has some known **limitations**:
 
 | Parameter Name | Type | Default Value | Possible Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
-| `displayName` | string |  |  | Optionsl. The friendly name of the management group. If no value is passed then this field will be set to the groupId. |
+| `displayName` | string |  |  | Optional. The friendly name of the management group. If no value is passed then this field will be set to the group ID. |
 | `name` | string |  |  | Required. The identifier of the management group |
```
Comment (MariusStorhaug) [inline arm/Microsoft.Management/managementGroups/readme.md:22]:
```suggestion
| `name` | string |  |  | Required. The group ID of the Management group |
```

File: constructs/Microsoft.Management/managementGroup-structure/readme.md [hunk 6]
```diff
@@ -63,9 +63,9 @@ Describes the Management groups to be created. Each management group is represen
 
 | Parameter Name | Type | Default Value | Possible values | Description |
 | :-             | :-   | :-            | :-              | :-          |
-| `name` | string | | | Mandatory. The ID of the Management group |
+| `name` | string | | | Mandatory. The resource ID of the Management group |
 | `parentId` | string | | A MG name | Mandatory. The template will concatenate `/providers/Microsoft.Management/managementGroups/` to create the resource ID of the parent management group the deployed one is child of |
```
Comment (MariusStorhaug) [inline constructs/Microsoft.Management/managementGroup-structure/readme.md:67]:
```suggestion
| `parentId` | string | | A MG name | Required. The template will concatenate `/providers/Microsoft.Management/managementGroups/` to create the resource ID of the parent management group the deployed one is child of |
```

File: arm/Microsoft.Management/managementGroups/readme.md [hunk 7]
```diff
@@ -76,8 +76,8 @@ If owner access is excessive, the following rights roles will grant enough right
 Consider using the following script:
 
 ```powershell
-$PrincipalID = "<The id of the identity here>"
-$TopMGID = "<The id of the management group here>"
+$PrincipalID = "<The resource ID of the identity here>"
+$TopMGID = "<The resource ID of the management group here>"
```
Comment (MariusStorhaug) [inline arm/Microsoft.Management/managementGroups/readme.md:80]:
```suggestion
$PrincipalID = "<The object ID of the identity here>"
$TopMGID = "<The group ID of the management group here>"
```

File: arm/Microsoft.Authorization/policyAssignments/deploy.bicep [hunk 8]
```diff
@@ -42,7 +42,7 @@ param enforcementMode string = 'Default'
 @description('Optional. The Target Scope for the Policy. The name of the management group for the policy assignment')
 param managementGroupId string = ''
 
-@description('Optional. The Target Scope for the Policy. The Id of the subscription for the policy assignment')
+@description('Optional. The Target Scope for the Policy. The resource ID of the subscription for the policy assignment')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/policyAssignments/deploy.bicep:45]:
```suggestion
@description('Optional. The Target Scope for the Policy. The subscription ID of the subscription for the policy assignment')
```

File: arm/Microsoft.Authorization/policyDefinitions/deploy.bicep [hunk 9]
```diff
@@ -29,10 +29,10 @@ param parameters object = {}
 @description('Required. The Policy Rule details for the Policy Definition')
 param policyRule object
 
-@description('Optional. The ID of the Management Group (Scope). Cannot be used with subscriptionId and does not support tenant level deployment (i.e. \'/\')')
+@description('Optional. The resource ID of the Management Group (Scope). Cannot be used with subscriptionId and does not support tenant level deployment (i.e. \'/\')')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/policyDefinitions/deploy.bicep:32]:
```suggestion
@description('Optional. The group ID of the Management Group (Scope). Cannot be used with subscriptionId and does not support tenant level deployment (i.e. \'/\')')
```

File: arm/Microsoft.Authorization/policyDefinitions/deploy.bicep [hunk 10]
```diff
@@ -29,10 +29,10 @@ param parameters object = {}
...
 param managementGroupId string = ''
 
-@description('Optional. The ID of the Azure Subscription (Scope). Cannot be used with managementGroupId')
+@description('Optional. The resource ID of the Azure Subscription (Scope). Cannot be used with managementGroupId')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/policyDefinitions/deploy.bicep:35]:
```suggestion
@description('Optional. The subscription ID of the subscription (Scope). Cannot be used with managementGroupId')
```

File: arm/Microsoft.Authorization/policyExemptions/deploy.bicep [hunk 11]
```diff
@@ -29,10 +29,10 @@ param policyDefinitionReferenceIds array = []
 @description('Optional. The expiration date and time (in UTC ISO 8601 format yyyy-MM-ddTHH:mm:ssZ) of the policy exemption. e.g. 2021-10-02T03:57:00.000Z ')
 param expiresOn string = ''
 
-@description('Optional. The ID of the management group to be exempted from the policy assignment. Cannot use with subscription id parameter.')
+@description('Optional. The resource ID of the management group to be exempted from the policy assignment. Cannot use with subscription ID parameter.')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/policyExemptions/deploy.bicep:32]:
```suggestion
@description('Optional. The group ID of the management group to be exempted from the policy assignment. Cannot use with subscription ID parameter.')
```

File: arm/Microsoft.Authorization/policyExemptions/deploy.bicep [hunk 12]
```diff
@@ -29,10 +29,10 @@ param policyDefinitionReferenceIds array = []
...
 param managementGroupId string = ''
 
-@description('Optional. The ID of the azure subscription to be exempted from the policy assignment. Cannot use with management group id parameter.')
+@description('Optional. The resource ID of the azure subscription to be exempted from the policy assignment. Cannot use with management group ID parameter.')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/policyExemptions/deploy.bicep:35]:
```suggestion
@description('Optional. The subscription ID of the subscription to be exempted from the policy assignment. Cannot use with management group ID parameter.')
```

File: arm/Microsoft.Authorization/policySetDefinitions/deploy.bicep [hunk 13]
```diff
@@ -10,13 +10,13 @@ param displayName string = ''
 @description('Optional. The Description name of the Set Definition (Initiative)')
 param policySetDescription string = ''
 
-@description('Optional. The ID of the Management Group (Scope). Cannot be used with subscriptionId and does not support tenant level deployment (i.e. \'/\')')
+@description('Optional. The resource ID of the Management Group (Scope). Cannot be used with subscriptionId and does not support tenant level deployment (i.e. \'/\')')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/policySetDefinitions/deploy.bicep:13]:
```suggestion
@description('Optional. The group ID of the Management Group (Scope). Cannot be used with subscriptionId and does not support tenant level deployment (i.e. \'/\')')
```

File: arm/Microsoft.Authorization/policySetDefinitions/deploy.bicep [hunk 14]
```diff
@@ -10,13 +10,13 @@ param displayName string = ''
...
 param managementGroupId string = ''
 
-@description('Optional. The ID of the Azure Subscription (Scope). Cannot be used with managementGroupId')
+@description('Optional. The resource ID of the Azure Subscription (Scope). Cannot be used with managementGroupId')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/policySetDefinitions/deploy.bicep:16]:
```suggestion
@description('Optional. The subscriptionID of the subscription (Scope). Cannot be used with managementGroupId')
```

File: arm/Microsoft.Authorization/roleDefinitions/deploy.bicep [hunk 15]
```diff
@@ -18,7 +18,7 @@ param dataActions array = []
 @description('Optional. List of denied data actions.')
 param notDataActions array = []
 
-@description('Optional. The ID of the Management Group where the Role Definition and Target Scope will be applied to. Cannot use when Subscription or Resource Groups Parameters are used.')
+@description('Optional. The resource ID of the Management Group where the Role Definition and Target Scope will be applied to. Cannot use when Subscription or Resource Groups Parameters are used.')
```
Comment (MariusStorhaug) [inline arm/Microsoft.Authorization/roleDefinitions/deploy.bicep:21]:
```suggestion
@description('Optional. The group ID of the Management Group where the Role Definition and Target Scope will be applied to. Cannot use when Subscription or Resource Groups Parameters are used.')
```

File: arm/Microsoft.Compute/virtualMachines/extensions/deploy.bicep [hunk 16]
```diff
@@ -62,7 +62,7 @@ resource extension 'Microsoft.Compute/virtualMachines/extensions@2021-07-01' = {
 @description('The name of the extension')
 output extensionName string = extension.name
 
-@description('The ResourceId of the extension')
+@description('The resource ID of the extension')
 output extensionId string = extension.id
```
Comment (MariusStorhaug) [inline arm/Microsoft.Compute/virtualMachines/extensions/deploy.bicep:66]:
```suggestion
output extensionResourceId string = extension.id
```

File: arm/Microsoft.EventHub/namespaces/disasterRecoveryConfigs/deploy.bicep [hunk 17]
```diff
@@ -4,10 +4,10 @@ param namespaceName string
 @description('Required. The name of the disaster recovery config')
 param name string
 
-@description('Optional. ARM Id of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
+@description('Optional. ARM ID of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
```
Comment (MariusStorhaug) [inline arm/Microsoft.EventHub/namespaces/disasterRecoveryConfigs/deploy.bicep:7]:
```suggestion
@description('Optional. Resource ID of the Primary/Secondary event hub namespace name, which is part of GEO DR pairing')
```

File: arm/Microsoft.Sql/servers/databases/deploy.bicep [hunk 18]
```diff
@@ -102,7 +102,7 @@ resource database 'Microsoft.Sql/servers/databases@2021-02-01-preview' = {
 @description('The name of the deployed database')
 output databaseName string = database.name
 
-@description('The resourceId of the deployed database')
+@description('The resource ID of the deployed database')
 output databaseId string = database.id
```
Comment (MariusStorhaug) [inline arm/Microsoft.Sql/servers/databases/deploy.bicep:106]:
```suggestion
output databaseResourceId string = database.id
```

File: arm/Microsoft.Sql/managedInstances/deploy.bicep [hunk 19]
```diff
@@ -60,10 +60,10 @@ param proxyOverride string = 'Proxy'
 @description('Optional. Whether or not the public data endpoint is enabled.')
 param publicDataEndpointEnabled bool = false
 
-@description('Optional. Id of the timezone. Allowed values are timezones supported by Windows.')
+@description('Optional. ID of the timezone. Allowed values are timezones supported by Windows.')
 param timezoneId string = 'UTC'
 
-@description('Optional. The Id of the instance pool this managed server belongs to.')
+@description('Optional. The resource ID of the instance pool this managed server belongs to.')
 param instancePoolId string = ''
```
Comment (MariusStorhaug) [inline arm/Microsoft.Sql/managedInstances/deploy.bicep:67]: Guess this should also reference ResourceID in the actual parameter? This needs an update to the Parameter file too.

```suggestion
param instancePoolResourceId string = ''
```

File: arm/Microsoft.ServiceBus/namespaces/migrationConfigurations/deploy.bicep [hunk 20]
```diff
@@ -9,10 +9,10 @@ param name string = '$default'
 @description('Required. Name to access Standard Namespace after migration')
 param postMigrationName string
 
-@description('Required. Existing premium Namespace ARM Id name which has no entities, will be used for migration')
+@description('Required. Existing premium Namespace ARM ID name which has no entities, will be used for migration')
 param targetNamespace string
```
Comment (MariusStorhaug) [inline arm/Microsoft.ServiceBus/namespaces/migrationConfigurations/deploy.bicep:13]: what is an "ARM ID name" ? Is this Resource ID? I also don't understand the sentence.

File: arm/Microsoft.ServiceBus/namespaces/disasterRecoveryConfigs/deploy.bicep [hunk 21]
```diff
@@ -9,10 +9,10 @@ param name string = 'default'
 @description('Optional. Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
 param alternateName string = ''
 
-@description('Optional. ARM Id of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
+@description('Optional. ARM ID of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
 param partnerNamespace string = ''
```
Comment (MariusStorhaug) [inline arm/Microsoft.ServiceBus/namespaces/disasterRecoveryConfigs/deploy.bicep:13]: ARM ID? Is this Resource ID? 
```suggestion
@description('Optional. Resource ID of the Primary/Secondary event hub namespace name, which is part of GEO DR pairing')
param partnerNamespaceResourceID string = ''
```

File: arm/Microsoft.OperationalInsights/workspaces/dataSources/deploy.bicep [hunk 22]
```diff
@@ -20,7 +20,7 @@ param kind string = 'AzureActivityLog'
 @description('Optional. Tags to configure in the resource.')
 param tags object = {}
 
-@description('Optional. Id of the resource to be linked.')
+@description('Optional. ID of the resource to be linked.')
```
Comment (MariusStorhaug) [inline arm/Microsoft.OperationalInsights/workspaces/dataSources/deploy.bicep:23]:
```suggestion
@description('Optional. Resource ID of the resource to be linked.')
```

File: arm/Microsoft.Network/virtualNetworkGateways/deploy.bicep [hunk 23]
```diff
@@ -7,7 +7,7 @@ param location string = resourceGroup().location
 @description('Optional. Specifies the name of the Public IP used by the Virtual Network Gateway. If it\'s not provided, a \'-pip\' suffix will be appended to the gateway\'s name.')
 param gatewayPipName array = []
 
-@description('Optional. Resource Id of the Public IP Prefix object. This is only needed if you want your Public IPs created in a PIP Prefix.')
+@description('Optional. Resource ID of the Public IP Prefix object. This is only needed if you want your Public IPs created in a PIP Prefix.')
 param publicIPPrefixId string = ''
```
Comment (MariusStorhaug) [inline arm/Microsoft.Network/virtualNetworkGateways/deploy.bicep:11]:
```suggestion
@description('Optional. Resource ID of the Public IP Prefix object. This is only needed if you want your Public IPs created in a PIP Prefix.')
param publicIPPrefixResourceId string = ''
```

File: arm/Microsoft.Network/virtualNetworkGateways/deploy.bicep [hunk 24]
```diff
@@ -48,7 +48,7 @@ param virtualNetworkGatewaySku string
 ])
 param vpnType string = 'RouteBased'
 
-@description('Required. Virtual Network resource Id')
+@description('Required. Virtual Network resource ID')
 param vNetId string
```
Comment (MariusStorhaug) [inline arm/Microsoft.Network/virtualNetworkGateways/deploy.bicep:52]:
```suggestion
@description('Required. Virtual Network resource ID')
param vNetResourceId string
```

File: arm/Microsoft.Network/publicIPAddresses/deploy.bicep [hunk 25]
```diff
@@ -1,7 +1,7 @@
 @description('Required. The name of the Public IP Address')
 param name string
 
-@description('Optional. Resource Id of the Public IP Prefix object. This is only needed if you want your Public IPs created in a PIP Prefix.')
+@description('Optional. Resource ID of the Public IP Prefix object. This is only needed if you want your Public IPs created in a PIP Prefix.')
 param publicIPPrefixId string = ''
```
Comment (MariusStorhaug) [inline arm/Microsoft.Network/publicIPAddresses/deploy.bicep:5]:
```suggestion
@description('Optional. Resource ID of the Public IP Prefix object. This is only needed if you want your Public IPs created in a PIP Prefix.')
param publicIPPrefixResourceId string = ''
```

File: arm/Microsoft.Network/privateEndpoints/privateDnsZoneGroups/deploy.bicep [hunk 26]
```diff
@@ -1,7 +1,7 @@
 @description('Required. The name of the private endpoint')
 param privateEndpointName string
 
-@description('Required. List of private DNS Ids')
+@description('Required. List of private DNS IDs')
 param privateDNSIds array
```
Comment (MariusStorhaug) [inline arm/Microsoft.Network/privateEndpoints/privateDnsZoneGroups/deploy.bicep:5]: Assuming these are not Resource IDs.

File: arm/Microsoft.Network/privateEndpoints/deploy.bicep [hunk 27]
```diff
@@ -1,10 +1,10 @@
 @description('Required. Name of the private endpoint resource to create.')
 param name string
 
-@description('Required. Resource Id of the subnet where the endpoint needs to be created.')
+@description('Required. Resource ID of the subnet where the endpoint needs to be created.')
 param targetSubnetId string
```
Comment (MariusStorhaug) [inline arm/Microsoft.Network/privateEndpoints/deploy.bicep:5]:
```suggestion
@description('Required. Resource ID of the subnet where the endpoint needs to be created.')
param targetSubnetResourceId string
```

**Review by MariusStorhaug — CHANGES_REQUESTED** [review]

File: arm/Microsoft.CognitiveServices/accounts/readme.md [hunk 28]
```diff
@@ -124,7 +124,7 @@ Tag names and tag values can be provided as needed. A tag can be left without a
     "defaultAction": "Deny",
     "virtualNetworkRules": [
       {
-        "id": "/subscriptions/<subscription-id>/resourceGroups/resourceGroup/providers/Microsoft.Network/virtualNetworks/<vnet-name>/subnets/<subnet-name>",
+        "ID": "/subscriptions/<subscription-ID>/resourceGroups/resourceGroup/providers/Microsoft.Network/virtualNetworks/<vnet-name>/subnets/<subnet-name>",
```
[the remark this answers is not in the document]
Comment (MrMCake) [reply arm/Microsoft.CognitiveServices/accounts/readme.md]: done

File: arm/Microsoft.DesktopVirtualization/hostpools/readme.md (hunk 1 above)
Comment (MrMCake) [reply arm/Microsoft.DesktopVirtualization/hostpools/readme.md]: done

File: arm/Microsoft.Automation/automationAccounts/softwareUpdateConfigurations/readme.md [hunk 29]
```diff
@@ -39,7 +39,7 @@ This module deploys an Azure Automation Account Software update Configuration.
 | `rebootSetting` | string |  | `[IfRequired, Never, RebootOnly, Always]` | Required. Reboot setting for the deployment schedule. |
 | `scheduleDescription` | string |  |  | Optional. The schedules description. |
 | `scopeByLocations` | array | `[]` |  | Optional. Specify locations to which to scope the deployment schedule to. |
-| `scopeByResources` | array | `[[subscription().id]]` |  | Optional. Specify the resources to scope the deployment schedule to. |
+| `scopeByResources` | array | `[[subscription().ID]]` |  | Optional. Specify the resources to scope the deployment schedule to. |
```
[the remark this answers is not in the document]
Comment (MrMCake) [reply arm/Microsoft.Automation/automationAccounts/softwareUpdateConfigurations/readme.md]: done

File: arm/Microsoft.Authorization/roleDefinitions/readme.md [hunk 30]
```diff
@@ -15,7 +15,7 @@ This module deploys custom RBAC Role Definitions.
 | `actions` | array | `[]` |  | Optional. List of allowed actions. |
 | `dataActions` | array | `[]` |  | Optional. List of allowed data actions. |
 | `location` | string | `[deployment().location]` |  | Optional. Location for all resources. |
-| `managementGroupId` | string |  |  | Optional. The ID of the Management Group where the Role Definition and Target Scope will be applied to. Cannot use when Subscription or Resource Groups Parameters are used. |
+| `managementGroupId` | string |  |  | Optional. The resource ID of the Management Group where the Role Definition and Target Scope will be applied to. Cannot use when Subscription or Resource Groups Parameters are used. |
```
[the remark this answers is not in the document]
Comment (MrMCake) [reply arm/Microsoft.Authorization/roleDefinitions/readme.md]: done

File: arm/Microsoft.Network/privateEndpoints/privateDnsZoneGroups/deploy.bicep (hunk 26 above)
Comment (MrMCake) [reply arm/Microsoft.Network/privateEndpoints/privateDnsZoneGroups/deploy.bicep]: check

File: arm/Microsoft.Authorization/roleDefinitions/deploy.bicep (hunk 15 above)
Comment (MrMCake) [reply arm/Microsoft.Authorization/roleDefinitions/deploy.bicep]: done

File: arm/Microsoft.ServiceBus/namespaces/migrationConfigurations/deploy.bicep (hunk 20 above)
Comment (MrMCake) [reply arm/Microsoft.ServiceBus/namespaces/migrationConfigurations/deploy.bicep:13]: That's a good question. It's actually from the official documentation

Comment (MrMCake) [reply arm/Microsoft.ServiceBus/namespaces/migrationConfigurations/deploy.bicep:13]: I guess it is a resource ID. Should we assume that?

Comment (MariusStorhaug) [reply arm/Microsoft.ServiceBus/namespaces/migrationConfigurations/deploy.bicep:13]: Lets assume that an ARM ID is indeed a resource ID :D

**Review by MariusStorhaug — APPROVED** [review]
Good to go...

**Pull request merged**